test_app/pubspec.yaml

```diff
@@ -25,6 +25,8 @@ dependencies:
   collection: ^1.16.0
   flutter_spinkit: ^5.1.0
   provider: ^6.0.0
+  chewie: ^1.3.6
+  video_player: ^2.2.18
```

Comment:
We need to get an OSS approval for any new package that we use. Can we implement this functionality without these?

Reply:
Chewie package is not that neccesary, we can do this without it. But video_player is needed.